pyproject.toml

```diff
@@ -24,9 +24,9 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^2.7,<2.8 || >= 3.5.0"
+python = ">= 3.6.0"
 distro = "^1.5.0"
-redis = "^3.5.3"
+redis = "^4.1.2"
 redis-py-cluster = "*"
```

Comment:
I think you want to remove redis-py-cluster here.

Reply:
Done